apps/bfd-server/bfd-server-war/pom.xml

```diff
@@ -404,6 +404,32 @@
 					</execution>
 				</executions>
 			</plugin>
+			<plugin>
+				<!-- Copy the key/trust stores over to target for easier access in the tests. -->
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-resources-plugin</artifactId>
+				<executions>
+					<execution>
+						<id>copy-server-resources</id>
+						<phase>generate-sources</phase>
+						<goals>
+							<goal>copy-resources</goal>
+						</goals>
+						<configuration>
+							<outputDirectory>${project.build.directory}/server-work</outputDirectory>
+							<resources>
+								<resource>
+									<directory>${basedir}/../dev/ssl-stores</directory>
+									<includes>
+										<include>server-keystore.jks</include>
+										<include>server-truststore.jks</include>
+									</includes>
```

Comment:
This copies a couple of the files needed for the server to start into target/server-work to facilitate finding them a bit easier; else we need to navigate to outside the module's scope and that's just messy.